### ISSUE TITLE
Build Touchkeys kit

### ISSUE THREAD

**Issue by pingdynasty** [RebelTechnology/FortRebel#43]
on Novation Impulse 61 keyboard, top octaves.
Will get remaining 3 octaves of parts from Andrew at a future date.

**Comment by padski** [comment]
Hello

**Comment by padski** [comment]
Done as required, lid left open, screws in bag

**Comment by padski** [comment]
Keyboard wiring etc.: Fit Touchkeys usb socket in rear panel. Fit MidiIO board in rear panel. Fit 2 x DPCO switches in rear panel. Wire existing keyboard usb power to touchkeys usb. Wire MidiIO TX/RX to keyboard TX/RX via switches.

**Comment by padski** [comment]
Awaiting hardware, ordered today 7-7-15

**Comment by padski** [comment]
Finished and boxed up

**Comment by ghost** [comment]
Todo: assemble next 3 octaves

**Comment by pingdynasty** [comment]
@padski are we missing parts? The bottom three octaves are not connected to control boards.

**Comment by padski** [comment]
Yes we need nice squidgy double-sided sticky things to replace the second-hand offcuts

**Comment by ghost** [comment]
but what about the control boards?

**Comment by pingdynasty** [comment]
sticky stuff: 3M 4658F, from Viking Tapes

**Comment by padski** [comment]
Very expensive are you sure? We only need a little.. can buy but will need to be a bit richer..:)
